test/basic.test.js

```diff
@@ -2,7 +2,7 @@
 const os = require('os')
 const { join } = require('path')
 const { readFileSync, existsSync, statSync } = require('fs')
-const { test } = require('tap')
+const { test, equal } = require('tap')
```

Comment:
avoid doing this, use the assertions that are passed into each test as they are contextualized

Reply:
✅